libcore/src/pedestrian/Pedestrian.cpp

```diff
@@ -1160,6 +1158,9 @@ bool Pedestrian::Relocate(std::function<void(const Pedestrian &)> flowupdater)
 #pragma omp critical(SetEgressTime)
                 allRooms.at(oldRoomID)->SetEgressTime(
                     GetGlobalTime()); //set Egresstime to old room //@todo: ar.graf : GetRoomID() yields NEW room
+#pragma omp critical(flowupdater)
+                flowupdater(
+                    *this); //@todo: ar.graf : this call should move into a critical region? check plz
```

Comment:
you don't want to remove this todo?

Reply:
The format looks pretty broken?

Reply:
Ah no, the comment should be behind ... anyways, ugly :D

Reply:
removed, and also changed the position of the other comment.